Classes/Api/ControlApi.php

```diff
@@ -63,7 +63,7 @@ public static function isOverwriteMode($infoArray)
         $overwriteMode = false;
         $conf = self::getConf();
 
-        $checkField = \JambageCom\TtProducts\Api\CustomerApi::getPossibleCheckField();
+        $checkField = CustomerApi::getPossibleCheckField();
```

Comment:
Has this change to transform the namespace been made manually or by a rector rule (which?) ?

Reply:
Maybe it is the rector rule \Rector\CodingStyle\Rector\Namespace_\ImportFullyQualifiedNamesRector::class like introduced here:

https://github.com/rectorphp/rector/issues/1117

But this is outdated.

Reply:
[Auto Import Names](https://github.com/sebpedley/rector/blob/main/docs/auto_import_names.md)

Reply:
Yes, that's the one. It's an option.

I used rector version `0.17.0` and for that version, this can be configured this way:

```php
return static function (RectorConfig $rectorConfig): void {
    // ...
    $rectorConfig->importNames(false, false);
    $rectorConfig->importShortClasses(false);
    // ...
};
```